### PR TITLE
Docs/review lm

### PR DESCRIPTION
**Pull request by lionelmoi**
My review is based on the analysis of different design systems: https://www.notion.so/lionelmoi/50cdf286eea74111aeac7f6e7604c8c2?v=97c13805755a4571af1c813169308206

I also based some modifications of the Microsoft style guide: https://docs.microsoft.com/en-us/style-guide/welcome/
Specifically https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/a/allow-allows

Explanations my changes below:

- demo/angularjs/components/button/doc.md
"allow" is almost never necessary. Reworked descriptions to remove it.
Harmonized descriptions of modes.
Clarified the Low emphasis description.

- demo/angularjs/components/checkbox/doc.md
Made an inline list for the enumeration (none, one or multiple)

- demo/angularjs/components/chip/doc.md
Harmonized formulation.

- demo/angularjs/components/dialog/doc.md
Split the first sentence and specified title and buttons as default.
Harmonized the descriptions

- demo/angularjs/components/dropdown/doc.md
Removed descriptions and modified titles to be more specific.

- demo/angularjs/components/list/doc.md
Simplified sentence

- demo/angularjs/components/progress-tracker/doc.md
moved the helper text mention to the main description
simplified the states description
clairified the error description

- demo/angularjs/components/radio-button/doc.md
simplified description

- demo/angularjs/components/select/doc.md
Avoided allow as it is not specific enough

- demo/angularjs/components/switch/doc.md
Simplified description

- demo/angularjs/components/table/doc.md
Simplified description, removed example as it should be in the demo

- demo/angularjs/components/text-field/doc.md
Avoid "let", simplified the description

- demo/angularjs/components/toolbar/doc.md
Simplified the description
Clarified the mention of three sections by describing the three sections

- demo/angularjs/components/user-block/doc.md
Aligned the description of sizes on the other desc of sizes
Typed Vertical and Extended as layouts
Clarified Extended layout.

### CODE REVIEW

File: demo/angularjs/components/button/doc.md [hunk 1]
```diff
@@ -1,28 +1,29 @@
...
 
-**Buttons allow to trigger actions with a single click.**
+**Buttons trigger actions with a single click.**
 
 ## High emphasis
 
-Allow users to quickly identify the main action of the current screen.
-We should limit one high emphasis button per view.
+Use high emphasis to help users to identify the main action.
+There should not be more than one high emphasis button in a view.
```
Comment (lionelmoi) [inline demo/angularjs/components/button/doc.md]: ajouter exemple du modal

File: demo/angularjs/components/chip/doc.md [hunk 2]
```diff
@@ -1,33 +1,34 @@
 # Chip
 
-**Use chips to display distinct input, attribute or action. They can allow users to make choices, enter information or filter content. Chips should always appear as the same action in the same general area.**
+**Chips display distinct inputs, attributes, or actions.**
+Use chips to display choices, as inputs or to filter content. All chips in the same area should represent the same action.
```
Comment (lionelmoi) [inline demo/angularjs/components/chip/doc.md]:
```suggestion

```

File: demo/angularjs/components/dialog/doc.md [hunk 3]
```diff
@@ -1,37 +1,40 @@
 # Dialog
 
-**Dialogs inform users about a task and can contain critical information, require decisions, or involve multiple tasks.**
+**Dialogs inform users and require an action.**
+Use dialogs to convey critical information or to request actions or decisions.
 
 ## Default
 
+Dialogs have a title, a body, and at one ore more buttons.
```
Comment (lionelmoi) [inline demo/angularjs/components/dialog/doc.md]:
```suggestion
Dialogs have a title, a body, and at least one ore more buttons.
```

File: demo/angularjs/components/dialog/doc.md [hunk 4]
```diff
@@ -1,37 +1,40 @@
 # Dialog
 
-**Dialogs inform users about a task and can contain critical information, require decisions, or involve multiple tasks.**
+**Dialogs inform users and require an action.**
```
Comment (lionelmoi) [inline demo/angularjs/components/dialog/doc.md]:
```suggestion
**Dialogs interrupts the users, gives information and require an action.**
```

File: demo/angularjs/components/dialog/doc.md [hunk 5]
```diff
@@ -1,37 +1,40 @@
...
 ## Default
 
+Dialogs have a title, a body, and at one ore more buttons.
+
 <demo-block component="dialog" partial="default" has-theme-switcher="false"></demo-block>
 
 ### With scrollbar
 
-When scrolling is required, the dialog title is pinned at the top, with buttons pinned at the bottom.
+When a dialog require scrolling, the title stays at the top while the buttons stay at the bottom.
```
Comment (lionelmoi) [inline demo/angularjs/components/dialog/doc.md]:
```suggestion
When a dialog requires scrolling, the title stays at the top while the buttons stay at the bottom.
```

File: demo/angularjs/components/dropdown/doc.md [hunk 6]
```diff
@@ -1,15 +1,12 @@
 # Dropdown
 
-**Use dropdown to present multiple actions in a small area. Dropdown are common used for contextual menus and in form selects.**
+**Dropdowns present multiple actions in a small area.**
+Dropdowns are commonly used for contextual menus and in form selects. They can contain dividers, icons, a user picture and a thumbnail.
 
-## Dropdown for menu
-
-Trigger a dropdown with a button. Dropdown can have divider, icons, user picture and thumbnail.
+## Open dropdown with a button
```
Comment (lionelmoi) [inline demo/angularjs/components/dropdown/doc.md]:
```suggestion

```

File: demo/angularjs/components/dropdown/doc.md [hunk 7]
```diff
@@ -1,15 +1,12 @@
...
 
 <demo-block component="dropdown" partial="default" has-theme-switcher="false"></demo-block>
 
 ## Open dropdown on target
```
Comment (lionelmoi) [inline demo/angularjs/components/dropdown/doc.md]:
```suggestion

```

File: demo/angularjs/components/progress-tracker/doc.md [hunk 8]
```diff
@@ -27,6 +27,6 @@ A step can be completed by a helper text.
 
 ### Error
 
-Error message replaces the helper text, if used.
+When an error occurs, the error message can replace the helper text. If there is no helper text, the error message is not displayed.
```
Comment (lionelmoi) [inline demo/angularjs/components/progress-tracker/doc.md]:
```suggestion
When an error occurs, the error message can replace the helper text.
```

File: demo/angularjs/components/radio-button/doc.md [hunk 9]
```diff
@@ -1,6 +1,6 @@
 # Radio button
 
-**Radio buttons are used to display multiple and exclusive options, meaning the user must select only one option.**
+**Radio buttons display exclusive options to chose from.**
```
Comment (lionelmoi) [inline demo/angularjs/components/radio-button/doc.md]:
```suggestion
**Radio buttons display exclusive options to chose from. Chose a default option.**
```

File: demo/angularjs/components/select/doc.md [hunk 10]
```diff
@@ -1,6 +1,6 @@
 # Select
 
-**Selects allow the user to make one or multiple choices.**
+**Selects display options in a dropdown list and present the users choice in the list box. Use it when users need to select one or more options.**
```
Comment (lionelmoi) [inline demo/angularjs/components/select/doc.md]:
```suggestion
**Selects display options in a list and present the users choice in the list box. Use it when users need to select one or more options.**
```

File: demo/angularjs/components/toolbar/doc.md [hunk 11]
```diff
@@ -1,10 +1,10 @@
 # Toolbar
 
-**The toolbar displays informations and actions relating to its current context.**
+**Toolbars display information and actions related to the context.**
 
 <demo-block component="toolbar" partial="default"></demo-block>
 
-The toolbar is divided in three section.<br />The title is placed to the left, while contextual actions are placed to the right.
+A toolbar has in three sections. An optional contextual icon button and a title on the left, contextual actions on the right.
```
Comment (lionelmoi) [inline demo/angularjs/components/toolbar/doc.md]:
```suggestion
A toolbar has three sections. An optional contextual icon button and a title on the left, contextual actions on the right.
```

File: demo/angularjs/components/user-block/doc.md [hunk 12]
```diff
@@ -1,9 +1,11 @@
 # User block
 
-**Displays user info.**
+**User block display user info.**
```
Comment (lionelmoi) [inline demo/angularjs/components/user-block/doc.md]:
```suggestion
**User blocks display user info.**
```

File: demo/angularjs/components/user-block/doc.md [hunk 13]
```diff
@@ -16,14 +18,20 @@
...
+There are two alternative layouts: _vertical_ and _extended_.
 
-Large user block can be displayed vertically.
+### Vertical
+
+A large user block can be displayed vertically.
 
 <demo-block component="user-block" partial="vertical"></demo-block>
 
 ## Extended
```
Comment (lionelmoi) [inline demo/angularjs/components/user-block/doc.md]:
```suggestion
### Extended
```

File: demo/angularjs/components/user-block/doc.md [hunk 14]
```diff
@@ -16,14 +18,20 @@
...
 
-## Vertical
+## Layouts
+
+There are two alternative layouts: _vertical_ and _extended_.
 
-Large user block can be displayed vertically.
+### Vertical
+
+A large user block can be displayed vertically.
```
Comment (lionelmoi) [inline demo/angularjs/components/user-block/doc.md]:
```suggestion
A _large_ size user block can be displayed vertically.
```

File: demo/angularjs/components/dialog/doc.md [hunk 15]
```diff
@@ -1,37 +1,40 @@
 # Dialog
 
-**Dialogs inform users about a task and can contain critical information, require decisions, or involve multiple tasks.**
+**Dialogs interrupts users, gives information and require an action.**
+Use dialogs to convey critical information or to request actions or decisions.
 
 ## Default
 
+Dialogs have a title, a body, and at least one ore more buttons.
```
Comment (lionelmoi) [inline demo/angularjs/components/dialog/doc.md]:
```suggestion
Dialogs have a title, a body, and at least one or more buttons.
```

File: demo/angularjs/components/dialog/doc.md [hunk 16]
```diff
@@ -1,37 +1,40 @@
 # Dialog
 
-**Dialogs inform users about a task and can contain critical information, require decisions, or involve multiple tasks.**
+**Dialogs interrupts users, gives information and require an action.**
```
Comment (FlorentDinet) [inline demo/angularjs/components/dialog/doc.md]: Dialogs interrupt

**Pull request merged**